crates/ironrdp-svc/src/lib.rs

```diff
@@ -162,10 +233,17 @@ pub trait StaticVirtualChannelProcessor: AsAny + fmt::Debug + Send {
         CompressionCondition::Never
     }
 
+    /// Start a channel, after the connection is established and the channel is joined.
+    ///
+    /// Returns a list of PDUs to be sent back.
+    fn start(&mut self) -> PduResult<Vec<SvcMessage>> {
+        Ok(vec![])
+    }
```

Comment:
> issue: As far as I understand, this method should only be called by the server. It would be good to document which function is intended to be called by the client and which is intended to be called by the server.

For all kind of channels, I am not sure client side doesn't send data first, or doesn't have to wait for the server initial message. Furthermore, with dynamic channels, you probably want to know when the channel is actually active. And with custom channels, there are even more unknown.

Reply:
> For all kind of channels, I am not sure client side doesn't send data first

Right, that’s true.

> Furthermore, with dynamic channels, you probably want to know when the channel is actually active.

This API is for static channels only. We’ll expose a different API for dynamic channels, because they are very different.

I would still suggest adding `SvcClientProcessor` and `SvcServerProcessor` traits, as marker traits then. This way we can encode at the type level which type may be used in which side and build other APIs on top of that (connector only takes `SvcClientProcessor`s and acceptor only takes `SvcServerProcessor`s).

Reply:
done